libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5778,6 +5779,16 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
             camera->send_camera_settings(chan);
         }
         break;
+    case MSG_CAMERA_FOV_STATUS:
```

Comment:
This should be protected with `AP_CAMERA_ENABLED`

Reply:
Its already.